### ISSUE TITLE
FEATURE REQUEST - Create shortcuts to open WSL Linux apps in Windows 11

### ISSUE THREAD

**Issue by diegocgaona** [Opticos/OpenInWSL-Source#10]
I was using WSLG in Windows 10 and it works great, and one very nice feature was create shortcuts to Linux apps.
In Windows 11 I'm not using it (but looks to work fine in Windows 11 too), I can open GUI apps, but I can't create shortcuts in my startmenu/desktop.

Would be great to have a app to do it or a feature.

Best regards, Diego.

**Comment by Pololot64** [comment]
You should be able to add shortcuts to the desktop and start menu. What have you been trying

**Comment by diegocgaona** [comment]
I'm uninstalled WSLG (because I don't use it because Windows 11 do not need it to GUI), and I don't want to use it only to create the shortcuts (because it will open using it as display).
I tried: https://wslutiliti.es/wslu/man/wslusc.html
But it didn't work for me (nothing happens after I use the shortcut). I'm using WSL2 with Ubuntu 20.04.
I'm trying to create a shortcut for Pycharm, it is installed with tar.gz, not apt (I installed Firefox with apt and Windows created a shortcut automatically).
